src/components/GroupBuildDetailPage/GroupBuildDetailPage.tsx

```diff
@@ -0,0 +1,116 @@
+  const serviceContainerGroupBuildRunner = serviceContainerGroupBuild.run;
+
+  const serviceContainerGroupBuildBuilds = useServiceContainer(groupBuildApi.getGroupBuildBuilds);
+  const serviceContainerGroupBuildBuildsRunner = serviceContainerGroupBuildBuilds.run;
+
+  useTitle(
+    generatePageTitle({
+      serviceContainer: serviceContainerGroupBuild,
+      firstLevelEntity: 'Group Build',
+      entityName: calculateLongBuildName(serviceContainerGroupBuild.data),
+    })
+  );
+
+  useEffect(() => {
+    serviceContainerGroupBuildRunner({ serviceData: { id: groupBuildId } });
+  }, [serviceContainerGroupBuildRunner, groupBuildId]);
+
+  useQueryParamsEffect(
+    ({ requestConfig } = {}) => serviceContainerGroupBuildBuildsRunner({ serviceData: { id: groupBuildId }, requestConfig }),
+    { componentId }
+  );
+
+  return (
+    <ServiceContainerLoading {...serviceContainerGroupBuild} title="Build details">
+      <PageLayout title={`Group Build ${calculateLongBuildName(serviceContainerGroupBuild.data)}`}>
+        <ContentBox padding marginBottom isResponsive>
+          <Attributes>
+            <AttributesItem title={groupBuildEntityAttributes.status.title}>
+              {serviceContainerGroupBuild.data?.status}
+            </AttributesItem>
+            <AttributesItem title={groupBuildEntityAttributes.groupConfig.title}>
+              {serviceContainerGroupBuild.data?.groupConfig.name}
+            </AttributesItem>
+            <AttributesItem title={groupBuildEntityAttributes['user.username'].title}>
+              {serviceContainerGroupBuild.data?.user.username}
+            </AttributesItem>
+            <AttributesItem title={groupBuildEntityAttributes.startTime.title}>
+              {serviceContainerGroupBuild.data?.startTime &&
+                createDateTime({ date: serviceContainerGroupBuild.data.startTime }).custom}
+            </AttributesItem>
+            <AttributesItem title={groupBuildEntityAttributes.endTime.title}>
+              {serviceContainerGroupBuild.data?.endTime &&
+                createDateTime({ date: serviceContainerGroupBuild.data.endTime }).custom}
+            </AttributesItem>
+          </Attributes>
+        </ContentBox>
+        <ContentBox borderTop>
+          <br />
+          <Toolbar disablePaddingTop>
+            <ToolbarItem>
+              <TextContent>
+                <Text component={TextVariants.h2}>Group Builds</Text>
+              </TextContent>
+            </ToolbarItem>
+            <ToolbarItem>
+              <Switch
+                id="toggle-build-tree-view"
+                label="Show Tree View"
+                isChecked={isTreeViewOn}
+                onChange={(checked) => {
+                  setIsTreeViewOn(checked);
+                }}
+              />
+            </ToolbarItem>
+          </Toolbar>
+          {isTreeViewOn && <DependencyTree groupBuild={serviceContainerGroupBuild.data} />}
+          {!isTreeViewOn && (
+            <BuildsList
```

Comment:
Currently, when the page is loaded, `BuildsList` data are fetched on the page load.
`DependencyTree` data are loaded when the component is displayed. Since `DependencyTree` handles its service container and the component is conditionally rendered, data are lost after the component is removed from the DOM.

Reply:
Hi @patrikk0123 
The calculation and logic of `DependencyTree` is a little bit complicated, and I want to decouple this module from the component that use it as much as possible. 
So do you prefer:

1. render the `DependencyTree` on load and find a way to hide/show it using the switch, or
2. Refactor the `DependencyTree` to pass the service container from the detail page into the component? (which I don't really like)

Reply:
Nested tabs will unify the behavior, yes.

Reply:
I have merged the PR #321.